src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx

```diff
@@ -39,7 +51,10 @@ export default class GuideDrawer extends React.Component {
         <StyledContent>
           <div
             dangerouslySetInnerHTML={{
-              __html: this.props.guide.steps[this.props.step - 1].message,
+              __html: this.interpolate(
```

Comment:
curious why not do this on the server, since we're just injecting html anyway

Reply:
We don't know the org slug at the time of API call (they could subsequently switch orgs client-side).